sway/sway.5.scd

```diff
@@ -725,12 +725,14 @@ The default colors are:
 *unbindswitch* <switch>:<state>
 	Removes a binding for when <switch> changes to <state>.
 
-*unbindsym* [--whole-window] [--border] [--exclude-titlebar] [--release] [--input-device=<device>] <key combo>
+*unbindsym* [--whole-window] [--border] [--exclude-titlebar] [--release] [--locked] \
+		[--to-code] [--input-device=<device>] <key combo>
 	Removes the binding for _key combo_ that was previously bound with the
 	given flags.  If _input-device_ is given, only the binding for that
 	input device will be unbound.
 
-	*unbindcode* [--whole-window] [--border] [--exclude-titlebar] [--release] [input-device=<device>] <code>
+	*unbindcode* [--whole-window] [--border] [--exclude-titlebar] [--release] \
+	[--locked] [input-device=<device>] <code>
```

Comment:
Style: missing indent here

Reply:
So the formatting in the man page is off here, but the style is the same as for bindcode description, following bindsym above.

Now that I take another look, and this may be a bug in scdoc, the current man page actually has a full tab-width gap between the [--release] and [--locked] args in bindcode - where the newline break and indent is in the markup. unbindcode used to be one line in the markup with fewer args so it doesn't currently have that formatting error.

You can get what I presume is the correct formatting with a _dedent_ following bindcode. It seems the tab is inserted literally there. Still looks kinda ugly in the markup though.

So now I've actually pushed the version with the corrected (?) formatting with the ugly dedent. Let me know what you think we should do here.